SPECS/kernel/kernel-uki.spec

```diff
@@ -0,0 +1,65 @@
+%global debug_package %{nil}
+
+# This should be a subpackage of the kernel package, but due to
+# "circular dependencies" that is not possible, so this is instead a
+# separate source package
+
+# Note - while Fedora's kernel version includes the %%_target_cpu as a
+# suffix, our kernel version does not.
+%define kernelver %{version}-%{release}
+
+%define cmdline console=ttyS0
+
+Summary:        Linux Kernel
+Name:           kernel-uki
+Version:        6.6.22.1
+Release:        2%{?dist}
```

Comment:
Why do we start with '2'? 

Generally, this is aligned with the kernel.spec. Is our intent to update this SPEC each time the kernel gets updated (version)?

Reply:
right, the %{release} is 2 here because the kernel.spec is also 2. this *should* be simply a subpackage of the normal `kernel.spec` but as stated in the `kernel-uki.spec` it can't be, due to "circular dependencies" that are a result of our build design (i.e. systemd BuildRequires: bpftool which is built by the kernel, and kernel-uki BuildRequires: systemd-ukify, so we can't put kernel-uki into the kernel.spec because it would create a "circular dependency" because we "build from scratch").